Apps/W1/HybridGP/app/src/codeunits/HybridGPManagement.codeunit.al

```diff
@@ -38,6 +38,18 @@ codeunit 4016 "Hybrid GP Management"
         CloseWizard := true;
     end;
 
+    [EventSubscriber(ObjectType::Table, Database::"Purchase Line", 'OnValidateQuantityOnBeforeCheckWithQuantityReceived', '', false, false)]
```

Comment:
So, we are taking ownership of these validations now? Assuming this is all the extra code added to the POMigrator. Any issues if\when BC changes validation stuff and we don't adjust for it? Not sure what that might be today, just trying to think ahead.

Reply:
This is required to even create the PO if the line has over receipt. The only alternative is to manipulate the numbers, which we decided against during a meeting.